### DISCUSSION TITLE
First Time Setup Query

### DISCUSSION THREAD

**Discussion by kennygokh** [Vinzent03/obsidian-git#603] [Q&A] [answered]
Hi, I have managed to setup obsidian git to backup my vault. May I know whether the initial setup does it upload all files in the vault to github repository? I am not able to see all files except some changes made in the vault that get uploaded. So I am not sure whether it is correctly setup. Please advise.

**Comment by Vinzent03** [comment, marked as answer]
You can see in the source control view which files are ready to be committed. Files that don't appear there are probably ignored in .gitignore or already committed. Verify that all commits are pushed to the remote by running the push command.
